app/javascript/groceries/components/StoreListEntry.vue

```diff
@@ -4,7 +4,7 @@
   @click="groceriesStore.selectStore({ store })"
 )
   .store-name
-    a {{ store.name }}
+    button {{ store.name }}
```

Comment:
This change caused a visual bug, fixed here: https://github.com/davidrunger/david_runger/pull/6691 .